src/m365/pp/base/PowerPlatformCommand.spec.ts

```diff
@@ -0,0 +1,130 @@
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+import appInsights from '../../../appInsights';
+import auth from '../../../Auth';
+import { Logger } from '../../../cli';
+import { CommandError } from '../../../Command';
+import Utils from '../../../Utils';
+import PowerPlatformCommand from './PowerPlatformCommand';
+
+class MockCommand extends PowerPlatformCommand {
+  public get name(): string {
+    return 'mock';
+  }
+
+  public get description(): string {
+    return 'Mock command';
+  }
+
+  public commandAction(logger: Logger, args: any, cb: () => void): void {
+    cb();
+  }
+
+  public commandHelp(): void {
+  }
+}
+
+describe('GraphCommand', () => {
+  before(() => {
+    sinon.stub(appInsights, 'trackEvent').callsFake(() => { });
+  });
+
+  afterEach(() => {
+    Utils.restore(auth.restoreAuth);
+  });
+
+  after(() => {
+    Utils.restore(appInsights.trackEvent);
+  });
+
+  it('correctly reports an error while restoring auth info', (done) => {
```

Comment:
Since we're not changing anything about auth or executing the command, do we need these tests?

Reply:
Assumed we needed that as it is part of the Graph one, but makes sense, removed both `correctly reports an error while restoring auth info` and `doesn\'t execute command when error occurred while restoring auth info`